projects/igniteui-angular/migrations/update-9_0_0/index.spec.ts

```diff
@@ -43,4 +43,32 @@ describe('Update 9.0.0', () => {
             `);
         done();
     });
+
+    it('should update Enum names.', done => {
+        appTree.create(
+            '/testSrc/appPrefix/component/enum.component.ts',
+            `import { AvatarType, Size, Type, SliderType } from 'igniteui-angular';
```

Comment:
Slight concern with the default migration for identifiers (class names, enums, etc).
Never assumed we'd be renaming something that can collide with the user's or 3rd party naming (unless the user manually created a shadowing `Igx-` class.
What I'm getting to is these might end up replacing an unrelated `Size` class for example as the migrations look for the identifiers globally.

Just FYI. If we think this may be too risky, let me know and I can log an issue and see if I can filter identifiers with import origin as well.

Reply:
Now that you say that, it will be risky. Type and Size in particular could very well be defined in an app. Let's make sure that only imports from igniteui-angular get renamed.

Reply:
@damyanpetev If we don't filter based on the import origin  the "Type" migration will replace one of angular core's classes, which we use in the angular-samples repo:
import { Type } from "@angular/core";
So with the current migration the samples would be broken.
Perhaps the migration logic should be updated first (if possible) before we merge this PR.

Reply:
Ah so it's actually very likely these to clash with something. Will log an addition for the base migrations to filter out just our imports before this can be merged. Thanks.